Package.swift

```diff
@@ -465,7 +465,7 @@ let package = Package(
         .package(url: "https://github.com/tuist/XcodeProj", exact: "8.19.0"),
         .package(url: "https://github.com/cpisciotta/xcbeautify", .upToNextMajor(from: "2.5.0")),
         .package(url: "https://github.com/krzysztofzablocki/Difference.git", from: "1.0.2"),
-        .package(url: "https://github.com/Kolos65/Mockable.git", from: "0.0.9"),
+        .package(url: "https://github.com/Kolos65/Mockable.git", exact: "0.0.10"),
```

Comment:
`MockableTest` was removed in `0.0.11`. Updating would mean a lot of changes unrelated to this PR, so the upgrade should be done in a follow-up.